cmd/tailscale/cli/up.go

```diff
@@ -465,6 +467,11 @@ func runUp(ctx context.Context, args []string) error {
 		backendState:  st.BackendState,
 		curExitNodeIP: exitNodeIP(curPrefs, st),
 	}
+
+	if upArgs.runSSH != curPrefs.RunSSH && isSSHOverTailscale() {
```

Comment:
True to false and false to true might be worth having different error texts for people. The false to true case might be counterintuitive for people (and actually we could just fix that with better TCP flow tracking)

Reply:
How's this?